src/_DataStructures_/Trees/BinaryTree/bottom-view-binary-tree/BottomViewBinaryTree.test.js

```diff
@@ -0,0 +1,17 @@
+const BinaryTree = require("../index");
+const bottomView = require('.');
+
+describe('Bottom View Binary Tree', () => {
+  let btree
+
+  beforeEach(() => {
+    btree = new BinaryTree([1, 2, 3, 4, 5, 6]);
+  });
+
+  it('Determines the bottom view of a binary tree', () => {
```

Comment:
'Determines the bottom view of a binary tree' -> `Should determines the bottom view of a binary tree`
It's all about Behavior Driven Testing, hence it starts with or contains should